client/version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "11.52-SNAPSHOT"
+version in ThisBuild := "11.52"
```

Comment:
this need to stay we use `sbt` `release` plugin that is handling this for use.

Reply:
Oh, sorry. I hadn't noticed that I committed this after trying to release locally. Thanks for catching it!